### ISSUE TITLE
Feature request: Add option to display percentage of "focussed" samples

### ISSUE THREAD

**Issue by alk** [gperftools/gperftools#291]
Originally reported on Google Code with ID 288

```
This is a feature request, not a bug report.

When I use "pprof --focus=<func>", I choose <func> as the entry point of one of my
threads.  This correctly limits the output graph to just the samples from that thread.
 But the percentages in the graphical display are still computed as a fraction of the
total number of samples, not the number of samples from that thread.

In other words, the reason I am using "--focus" is to see what fraction of a specific
thread's time is consumed by various functions, but the graphical output shows me the
(relatively uninteresting) percentage that I get when I do not use "--focus".  Of course
I can do the calculation in my head, but it would be nice if pprof could do it for
me as an option.

Thanks!

```

Reported by `lopresti` on 2010-12-01 19:01:38
